README.md

```diff
@@ -44,7 +44,15 @@ import connectToChild from 'penpal/lib/connectToChild';
 
 const iframe = document.createElement('iframe');
 iframe.src = 'http://example.com/iframe.html';
-document.body.appendChild(iframe);
+if (document.readyState === "complete"
+  || document.readyState === "loaded"
```

Comment:
Thanks @corollari. When are you seeing a `readyState` value of `loaded`?

Reply:
[This stackoverflow answer](https://stackoverflow.com/a/9457996) claims that this is the readiness state used in older versions of safari, thus we're getting backwards compatibility with those by including it here. 

But, given that this snippet will be in the readme, and the percentage of old safari versions in the wild is a minuscule part of total browser marketshare I can also understand arguments for removing it.